### DISCUSSION TITLE
Software features that will put Hi.Events up against the other guys

### DISCUSSION THREAD

**Discussion by jxs714** [HiEventsDev/Hi.Events#104] [Ideas]
So far the software looks great! I am excited to see the growth of your software and hopefully see features such as

- Memberships, merch and different types of tickets and add-ons
- Event mapping and vendor management
- Custom emails for purchases
- Possibly an email builder similar to mailchimp and a way to create your mailing list and send messages to ticket buyers
- Seating
- ios and android app for scanning tickets and maybe managing the event
- Custom background image
- Video such as youtube embedded
- Google pay and apple pay with stripe
- Google and Apple wallet with tickets and option for you to customize graphics and info

Hopefully you guys can take a look at some of the big guys and replicate features from platform such as Sh0wclix, Ev3nt3ny, Tick3tsp1c3 and others. You guys would do insanely well and will get people to come over to purchase a license. I 100% would purchase a license if it were as affordable as it is now with these features, especially if vendor management and mapping is added! I would even go ahead and donate to the project on top of the license purchase because it would be so worth it.

Keep up the great work!

**Comment by daveearley** [comment]
Great list of features @jxs714! A lot of them are on the roadmap, especially seating. A lot of the homepage customisation should be possible now, and the other features will be implemented soon.

Customisation is the main focus. Over the next few months you'll see a lot of new features in that area.

Thank,
Dave

**Reply by justynpride** [reply]
I'd +1 to the ability for add ons onto tickets that impact the ticket price - likely to be Paid add ons such as seminars, meal tickets, or even accommodation options. These you want linked to a specific attendee and ideally viewable on the ticket and confirmation email. 

Of course this can be achieved as separate tickets already, but it's not as tidy unless tickets can be grouped or presented as different stepped pages in the order process (ie step one event tickets, step two merch/add ons, step three order review and payment). 

I'm sure there are a variety and of needs and solutions.

**Comment by islandmix** [comment]
Just curious... can you provide an example of what "vendor management" would look like? I want to confirm if I am thinking the same thing.

Thanks

> So far the software looks great! I am excited to see the growth of your software and hopefully see features such as
> 
> * Memberships, merch and different types of tickets and add-ons
> * Event mapping and vendor management

**Reply by jxs714** [reply]
Hi

Vendor management with features such as:

- An application approval system for vendors to apply and charge when approved
- Create an event map and placing vendors on the map and linking them to the paid vendor space when they purchase their space.
- Providing them tickets and entry qr codes to redeem their entry
- easily manage and print badges for the vendors
